src/test/java/com/google/step/finscholar/firebase/FirebaseStorageManagerTest.java

```diff
@@ -0,0 +1,47 @@
+package com.google.step.finscholar.firebase;
+
+import com.google.cloud.firestore.Firestore;
+import com.google.firebase.cloud.FirestoreClient;
+import com.google.step.finscholar.firebase.TestObject;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class FirebaseStorageManagerTest {
+  // Constants for FirebaseStorageManagerTest.
+  public static final String TEST_COLLECTION_NAME = "testObjects";
+  public static final String TEST_DOCUMENT_NAME = "testDocument";
+  public static final String EXPECTED_DOCUMENT_RETRIEVABLE = "{\"one\":\"testObjects\",\"two\":\"testDocument\"}";
+  private static Firestore firebase;
+  private static TestObject testObject;
+
+  @BeforeClass
+  public static void setUp() {
+    try {
+      firebase = FirestoreClient.getFirestore(FirebaseAppManager.getApp());
+    } catch (Exception e) {
+      System.out.println(e);
+    }
+    testObject = new TestObject(TEST_COLLECTION_NAME, TEST_DOCUMENT_NAME);
+  }
+
+  @Test
+  public void isRecentlySetDocumentRetrievable() {
+    try {
+      FirebaseStorageManager.storeDocument(firebase, TEST_COLLECTION_NAME, testObject, TEST_DOCUMENT_NAME);
+    } catch (Exception e) {
+      System.out.println(e.toString());
+    }
+    String json = "";
+    try {
```

Comment:
Sorry I didn't catch this earlier - you don't need to try/catch in test classes in java. You can just add a `throws Exception` clause after the method signature.

Reply:
Oh awesome! Thanks!